### ISSUE TITLE
Can you increase group control?

### ISSUE THREAD

**Issue by Helaer** [Genymobile/scrcpy#2465]
Can group control be added to realize mirroring of one device operation to multiple devices?

**Comment by rom1v** [comment]
No, scrcpy does not support that, and is too generic to be properly supported (the devices might be different, of different sizes, etc).
